verify-docker-versions

```diff
@@ -24,10 +24,20 @@ compare_versions() {
   return 0
 }
 
+# test that user has rights to use docker
+
+DOCKER_ENGINE_INFO_STDERR=$(docker info 2>&1 >/dev/null)
+if [[ -n "$(echo $DOCKER_ENGINE_INFO_STDERR | grep 'permission denied')" ]]; then
+  >&2 echo "Local user has insufficient permissions to connect to the docker daemon."
```

Comment:
Btw, it just occurred to me that this won't work in any non-English locale. Is there a better way we can detect this? Is simply return code enough? `docker info` should never fail otherwise right?

Reply:
You are right! There are other parts of the script that use docker info output, though.

I don't know if there are other reasons for `docker info` to fail, that is why I originally decided on using the stderr output

Anyway, fixed now. Have a new look and I will merge